core-web/libs/sdk/experiments/src/lib/sdk-experiments.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Represents an SDK Experiment.
+ * @class
+ */
+export class sdkExperiment {
```

Comment:
use PascalCase for class names

Reply:
good catch. Let me check if I can setup `eslint` to force that always.